code/modules/clothing/masks/gasmask.dm

```diff
@@ -85,6 +139,7 @@
 	actions_types = list(/datum/action/item_action/adjust)
 	dog_fashion = /datum/dog_fashion/head/clown
 	species_exception = list(/datum/species/golem/bananium)
+	max_filters = 0
```

Comment:
```suggestion
	max_filters = 1
```
If you're going to give the plague doctor mask a filter slot, don't deprive the mime and clown masks of the same thing. We made the clown and mime masks work with internals because we don't want them to have to remove their masks. This is an intentional design choice.